Source/Core/Core/NetworkCaptureLogger.h

```diff
@@ -43,6 +44,8 @@ class NetworkCaptureLogger
   NetworkCaptureLogger& operator=(NetworkCaptureLogger&&) = delete;
   virtual ~NetworkCaptureLogger();
 
+  virtual void LogSocket(s32 socket) = 0;
```

Comment:
I don't think this function name reflects what PCAPSSLCaptureLogger::LogSocket actually does. I'm bad at naming, so I'm not sure what to suggest, but maybe "OnSocketAdded" which would IMO make it clearer that you *must* call this function when a socket is added for things to work properly? or register socket, I don't know

Reply:
Done. I chose `OnNewSocket` regarding the naming. The `AddSocket` function is just a helper used in the `NewSocket` and when `accept` returns a new socket.